appveyor.yml

```diff
@@ -1,11 +1,11 @@
 clone_depth: 5
 environment:
-  nodejs_version: "10.15"
+  nodejs_version: "12.18"
```

Comment:
I would suggest to use "12" simply here. Then `Get-NodeJsLatestBuild` below should always get latest v12 version (in our case v12.18).

An alternative is that we could avoid `Get-NodeJsLatestBuild` at all. But by using it, we'll get warned of problems in newer versions of node v12, which is a nice thing. Because appveyor isn't a required CI for our project it's OK if this doesn't pass sometimes because of this.
On the contrary, sticking on a specific version for CircleCi is useful because it's a required CI so it makes sense that we control more strictly the environment there.

Reply:
Hi @julienw , thanks for the comments!

I came across that issue with JSDOM and TextEncoder/TextDecoder also and was just about to mention it. Looks like it will have to be left as it is for now! It did throw me for a while when nothing I tried was working!

I'll update the AppVeyor conf to just use the major version number, but I'll leave the CircleCi as major.minor like you said.